packages/synapse-react-client/src/components/JSONArrayEditor/JSONArrayEditor.tsx

```diff
@@ -99,38 +88,22 @@ function JSONArrayEditor<T = unknown>(props: JSONArrayEditorProps<T>) {
         },
       }}
     >
-      <Form
-        ref={formRef}
+      <JsonSchemaForm
+        formRef={formRef}
```

Comment:
Created a new component `JsonSchemaForm` which automatically incorporates all of our custom templates and widgets that are appropriate for a generic form UI.